src/compat/eme/custom_media_keys/webkit_media_keys.ts

```diff
@@ -114,7 +114,7 @@ class WebkitMediaKeySession
     this.keyStatuses = new Map();
     this.expiration = NaN;
 
-    this.update = (license: Uint8Array) => {
+    this.update = (license: Int8Array | ArrayBuffer) => {
```

Comment:
:warning: Int8Array != Uint8Array
One contains signed integers the other not

Also, I'm under the impression that the current implementation allows every ArrayBuffer + ArrayBufferView so every `BufferSource`, no need to be picky here.

Reply:
ho yes, VSCode autocompletion got me here :(

but yes, Typescript seems to be lost, because he doesn't know the type anymore after checking:

```
license instanceof Uint8Array
``` 

Actually, its true, what could be the type if the method waits for a `Uint8Array` but we check for it and it's not?